src/components/AssetInput/MultiAssetPreview/Assets.tsx

```diff
@@ -0,0 +1,45 @@
+import { merge } from "@utilities/merge";
+import React, { FC } from "react";
+import { AssetInputSize } from "../AssetInput";
+import { AssetThumbnail } from "../AssetThumbnail";
+import { MultiAssetPreviewProps } from "./MultiAssetPreview";
+
+export const Assets: FC<Pick<MultiAssetPreviewProps, "assets">> = ({ assets }) => {
+    const length = assets?.length || 0;
+    const assetsNew = assets?.slice(0, 4) || [];
+
+    if (length < 4) {
+        [...Array(4 - length)].map((item, index) =>
+            assetsNew.push({
+                name: `empty picture ${index + 1}`,
+                size: 20,
+                type: "image",
+                extension: "",
+                src: "",
+                source: "upload",
+            }),
+        );
+    }
+    return (
+        <div className="tw-border-black-20 tw-grid tw-grid-cols-2 tw-gap-0.5">
+            {assetsNew.map((asset, index) => {
+                console.log(asset);
```

Comment:
If you remove this you can make a direct return :)

Reply:
done